src/platforms/common/configuration/options.mdx

```diff
@@ -628,3 +627,17 @@ Set this callback, which is called after the Sentry React Native SDK initializes
 </ConfigKey>
 
 </PlatformSection>
+
+<PlatformSection supported={["flutter"]}>
+
+## Hybrid SDK Options
+
+<ConfigKey name="autoInitializeNativeSdk">
```

Comment:
There's already an `autoInitializeNativeSdk` key for React Native, maybe adapt the text in a generic way so it works for Flutter and React Native.

Reply:
RN mentions the `enableNative` option, which we don't have on flutter and we also link to different endpoints to setup manually, so it's different enough i guess?

Reply:
So my goal was to actually add this phrase:

> Do not use this to disable the native layer, but instead use [enableNative](#enableNative).

into a `<PlatformSection supported={["react-native"]}>` only, so the explanation of the `autoInitializeNativeSdk` section would work for RN and Flutter, what do you think?

Reply:
Check the `attach-screenshot` flag, the `new in version X` is only visible for Android, the idea would be the same.

Reply:
Hope i did this correctly.